src/idl_gen_dart.cpp

```diff
@@ -85,6 +85,27 @@ class DartGenerator : public BaseGenerator {
       : BaseGenerator(parser, path, file_name, "", ".", "dart"),
         namer_(WithFlagOptions(DartDefaultConfig(), parser.opts, path),
                DartKeywords()) {}
+
+  template<typename T>
+  void import_generateor(const std::vector<T *> &definitions,
```

Comment:
nit: `import_generator` here and elsewhere

Reply:
Thank you @dnfield , you are the best :)